Detectors/ITSMFT/common/simulation/include/ITSMFTSimulation/AlpideSimResponse.h

```diff
@@ -104,9 +107,9 @@ class AlpideSimResponse
   float getYMax() const { return mYMax; }
   float getZMin() const { return mZMin; }
   float getZMax() const { return mZMax; }
-  float getStepX() const { return mStepInvX ? mStepInvX : 0.f; }
-  float getStepY() const { return mStepInvY ? mStepInvY : 0.f; }
-  float getStepZ() const { return mStepInvZ ? mStepInvZ : 0.f; }
+  float getStepX() const { return mStepInvX ? 1./mStepInvX : 0.f; }
```

Comment:
would it be reasonable storing `mStepX` directly as a member? (avoids division)

Reply:
It is not used in any algorithm loop, just a method to know the segmentation (if needed).

Reply:
fair enough